README.md

```diff
@@ -115,6 +115,41 @@ uvx RepoAuditor --include GitHub \
 
 `RepoAuditor` will generate a series of messages describing all the issues in the repository, along with the rationale behind them and the steps for resolution.
 
+### Custom Settings
+
+`RepoAuditor` tries to provide default settings and checks which are considered best practices. However, your team or organization may have different ways of doing things.
+You can override the default settings by specifying the appropriate flag.
+
+We have 3 types of flags:
+
+1. Boolean flags for those settings which are checked to be on by default and you wish to check are off, with the form `Module-no-Requirement` (e.g. `--GitHub-no-MergeCommit`).
+2. Conversely, boolean flags for those settings which are checked to bed off by default, and you wish to check are on, denoted by the form `Module-Requirement` (e.g. `--GitHub-AllowDeletions`).

```

Comment:
"which are checked to bed off by default" -> "which are checked to be off by default"